src/pawelak-arek/Fsd.Arek.Client.Ex2/quizView.js

```diff
@@ -0,0 +1,32 @@
+var quizView = quizView || {};
+
+(function () {
+
+    var showQuestion = function (questionModel) {
+        quizWindow.question.html(questionModel);
+    }
+
+    var showAnswer = function (questionModel) {
+        if ($('.answer-box').length != 0) {
+            $('.answer-box').remove();
+        }
+
+        for (var i = 0; i < questionModel.length; i++) {
+            quizWindow.answerContainer.append($("<div>").addClass('answer-box').html(questionModel[i]).attr("onClick", "app.quizController.checkAnswer(event)"));
+        }
+    }
+
+    var showScore = function (text) {
+        $('.answer-box').remove();
```

Comment:
jeżeli dobrze pamiętam w tym zadaniu nie wolno używać jquery

Reply:
Na ostatnim zjeździe spytałem się czy możemy użyć JQ do tego zadania :) powiedziałeś, że nie gra różnicy ma działać :)   Postaram się zrobić to również w JS.

Reply:
ok, to nie jest istotne. W sb uczymy sie angulara i wtedy nawet przez sekunde nie pomyślisz o jQuery :)